frontend/src/components/keyboard/index.js

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react'
+import { useStore } from '../../store'
+import {default as SimpleKeyboard} from 'react-simple-keyboard'
+import 'react-simple-keyboard/build/css/index.css'
+import './style.css'
+
+const Keyboard = () => {
+    const {publishKey} = useStore()
+    const [layoutName, setLayoutName] = useState('default')
+
+    const onKeyPress = (button) => {
+        if (button === '{capslock}') {
+            setLayoutName(layoutName === 'default' ? 'shift' : 'default')
+        }
+        publishKey(button)
+    }
+
+    return (
+        <div className='keyboard-wrapper'>
+            <SimpleKeyboard
+                layoutName={layoutName}
+                onKeyPress={onKeyPress}
+                theme={'hg-theme-default custom-theme-dark'}
+                baseClass={'simple-keyboard-main'}
+                physicalKeyboardHighlight={true}
+                syncInstanceInputs={true}
+                mergeDisplay={true}
+                layout={{
+                    default: [
```

Comment:
Let's probably extract these layout and display settings from the component into constants or separate keyboard configs.

Reply:
done